submissions/lalalilulu/hooli-style_popup/styles.css

```diff
@@ -0,0 +1,166 @@
+@import url(https://fonts.googleapis.com/css?family=Roboto:wght@300&display=swap);
+
+nav {
```

Comment:
Avoid styling by id.
Separate responsibilities:
- HTML for markup
- Classes for styling
- Id for JS and for linking label to the interractive elements

Reply:
My bad, I forget to add HTML tag. avoid styling by HTML tag too.)